ContextDesktop/src/main/java/com/github/migi_1/Context/obstacle/ObstacleSpawner.java

```diff
@@ -42,6 +42,16 @@ public ObstacleSpawner(Commander commander) {
      * @return Map with all obstacles, with as key value their Geometry in the environment.
      */
     public ArrayList<Obstacle> getObstacles() {
+
+        //call removeDamageDealer when an obstacle is too far away
+        ArrayList<Obstacle> deleteList = new ArrayList<Obstacle>();
+        for (Obstacle obs : obstacleList) {
+            if ((obs.getModel().getLocalTranslation().x - commander.getModel().getLocalTranslation().x) > 200) {
+                deleteList.add(obs);
+
+            }
+        }
+        obstacleList.removeAll(deleteList);
```

Comment:
This method return the most up to date list of obstacles. Do you want me to rename or change it?